package.json

```diff
@@ -53,10 +52,12 @@
     "eslint-config-prettier": "^8.9.0",
     "eslint-plugin-jest": "^27.2.3",
     "husky": "^8.0.3",
+    "jest": "^29.6.4",
     "lint-staged": "^14.0.0",
     "mongodb-memory-server": "^8.15.1",
     "nodemon": "^3.0.1",
     "prettier": "^3.0.0",
+    "socket.io-client": "^4.7.2",
```

Comment:
We probably only need the client socket library in test mode as well; since in the main app, we only are currently running the socket in server-mode.

We can move this line to the `devDependencies` block..

Reply:
Akos , it is already in devDependencies block 
<img width="566" alt="image" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/88336329/dcc6cdee-4041-4851-8131-8b5dbe930892">